docs/pages/dialogs.rst

```diff
@@ -64,6 +64,30 @@ selection.
 .. image:: ../images/dialogs/confirm.png
 
 
+Button dialog
+--------------------------
+
+The :func:`~prompt_toolkit.shortcuts.dialogs.button_dialog` function displays a
+dialog with choices offered as buttons. Buttons are indicated as a tuple of
+tuples, providing the label (first) and return value if clicked (second).
+
+.. code:: python
+
+    from prompt_toolkit.shortcuts.dialogs import button_dialog
+
+    result = button_dialog(
+        title='Button dialog example',
+        text='Do you want to confirm?',
+        buttons=(
```

Comment:
Could you make this a list of tuples? 
See also this page for more about the difference: https://nedbatchelder.com/blog/201608/lists_vs_tuples.html

Also, I like spaces after the commas in the tuples. ;)